react/features/mobile/picture-in-picture/actions.js

```diff
@@ -21,13 +22,7 @@ import {
  */
 export function enterPictureInPicture() {
     return (dispatch: Dispatch, getState: Function) => {
-        const state = getState();
-        const { app } = state['features/app'];
-        const { conference, joining } = state['features/base/conference'];
-
-        if (app
-                && app.props.pictureInPictureEnabled
-                && (conference || joining)) {
+        if (isItOkToGoPiP(getState())) {
```

Comment:
I wanted to comment on the fact that we're dispatching "ENTER_PICTURE_IN_PICTURE" event which then is propagated to the external API. What's the point of doing that in case the native PictureInPicture would succeed with bring the app into the PiP mode. Wouldn't that duplicate the "enter picture in picture" call on the native side in case external API user has decided to handle this action (on Android) ? Or does this event only take effect on the iOS (which would be weird) ?

Reply:
@saghul what about this comment ?